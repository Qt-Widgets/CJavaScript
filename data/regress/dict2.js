dog = {color: "brown", size: "large"};
console.log(dog);

console.log(dog["color"]); // results in "brown"
console.log(dog.color);    // also results in "brown"

cats = [{color: "brown", size: "large"}, {color: "black", size: "small"}];
console.log(cats);

console.log(cats[0]["size"]); // results in "large"

dogs = {rover: {color: "brown", size: "large"}, spot: {color: "black", size: "small"}};
console.log(dogs);

console.log(dogs["spot"]["size"]); // results in "small"
console.log(dogs.rover.color);     // results in "brown"
