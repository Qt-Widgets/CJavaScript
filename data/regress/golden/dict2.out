{ color: 'brown', size: 'large' }
brown
brown
[ { color: 'brown', size: 'large' },
  { color: 'black', size: 'small' } ]
large
{ rover: { color: 'brown', size: 'large' },
  spot: { color: 'black', size: 'small' } }
small
brown
